system/Drivers/STM32U5xx_HAL_Driver/Inc/stm32u5xx_hal_conf_template.h

```diff
@@ -277,6 +286,10 @@ vary depending on the variations in voltage and temperature.*/
 #include "stm32u5xx_hal_dma2d.h"
 #endif /* HAL_DMA2D_MODULE_ENABLED */
 
+#ifdef HAL_DSI_MODULE_ENABLED
```

Comment:
https://github.com/stm32duino/Arduino_Core_STM32/blob/main/system/STM32U5xx/stm32u5xx_hal_conf_default.h
should be updated to add all the new HAL modules.

Reply:
Done